src/commands/junit/__tests__/upload.test.ts

```diff
@@ -34,9 +34,9 @@ describe('upload', () => {
     test('should read all xml files and reject invalid ones', async () => {
       const context = createMockContext()
       const command = new UploadJUnitXMLCommand()
-      const [firstFile, secondFile] = await command['getMatchingJUnitXMLFiles'].call(
+      const files = await command['getMatchingJUnitXMLFiles'].call(
         {
-          basePaths: ['./src/commands/junit/__tests__/fixtures'],
+          basePaths: ['src/commands/junit/__tests__/fixtures'],
```

Comment:
will it work with `./src...` or that's no longer valid?

Reply:
`./src` is still valid, but then the outputs will also be prefixed with `./` (which only matters in tests, as far as I can tell)